mitodl/open-discussions
# Added free prices to videos

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2402 

#### What's this PR do?
- Adds videos under the price and availability facets (free and available now, respectively)
- Fixes a flaky test that failed travis tests for this branch (second commit)

#### How should this be manually tested?

- Import some video data (doesn't have to be everything, the MITx channel is small/quick to import: `./manage.py backpopulate_youtube_data fetch -c UCTBMWu8yshnAmpzR3MoJFtw`)
- Verify those videos are now searchable under the facets described above and the results display with "Free" and "Available Now" accordingly (see screenshots)

#### Screenshots (if appropriate)

![Screenshot_2019-11-22 MIT Open Learning](https://user-images.githubusercontent.com/28598/69438371-e7568180-0d12-11ea-9523-f18d99913acc.png)
